docs/configuration/snap.md

```diff
@@ -79,7 +79,7 @@ The top-level [snap](configuration.md#Configuration-snap) key contains set of op
 <p>Inherited from <code>TargetSpecificOptions</code>:</p>
 <ul>
 <li><code id="SnapOptions-artifactName">artifactName</code> String | “undefined” - The <a href="/configuration/configuration#artifact-file-name-template">artifact file name template</a>.</li>
-<li><code id="SnapOptions-publish">publish</code> The <a href="/configuration/publish">publish</a> options.</li>
+<li><code id="SnapOptions-publish">publish</code> The <a href="/configuration/publish">publish</a> options.</li> To publish directly to Snapcraft, see <a href="https://snapcraft.io/docs/snapcraft-authentication">Snapcraft authentication options</a> for local or CI/CD authentication options.
```

Comment:
This is actually an auto-generated doc. You'll want to add the text here:
https://github.com/electron-userland/electron-builder/blob/2ca621289e025a6d0856c7c588aaac40d267cb5f/packages/builder-util-runtime/src/publishOptions.ts#L252C1-L252C1
Then run `pnpm generate-all` to update the docs.